scripts/bespoke_input.py

```diff
@@ -0,0 +1,101 @@
+#! /usr/local/bin/python
+
+import json
+import sys
+
+SRC = 1
+TGT = 2
+IP_PREF = 0
+TIME = 3
+
+class Update:
+    """
+    Internal representation of an individual BGP Update or Delta-net Update
+    """
+
+    def __init__(self, update_str):
+        """
+        Parses a Logical Rule Update from a delta-net string
+
+        Expected Format is
+        +<subnet_prefix>,<source_ip>,<next_hop_ip>,<timestamp>
+        or
+        -<subnet_prefix>,<source_ip>,*,<timestamp>
+        """
+        self.is_widthdraw = update_str[0] == "-"
+        fields = update_str[1:].strip().split(",")
+        assert(len(fields) == 4)
+        self.source = fields[SRC]
+        self.target = fields[TGT]
+        assert(self.is_widthdraw == (self.target == "*"))
+        self.ip_prefix = fields[IP_PREF]
+        self.timestamp = fields[TIME]
+        
+    
+    def to_BGP_string(self):
+        """
+        Returns a string representing the rule update in GoBGP format
+        """
+        if self.is_widthdraw:
+            bgp = {"Header": {"Type" : 0},
+                   "PeerHeader" : {"PeerBGPID" : self.source, "Timestamp" : int(self.timestamp)},
+                   "Body" : {
+                       "BGPUpdate" : {
+                           "Body" : {
+                               "PathAttributes" : [],
+                               "NLRI" : [],
+                               "WithdrawnRoutes" : [{"prefix" : self.ip_prefix}]
+                           }
+                       }
+                   }
+            }
+        else:
+            bgp = {"Header": {"Type" : 0},
+                   "PeerHeader" : {"PeerBGPID" : self.source, "Timestamp" : int(self.timestamp)},
```

Comment:
As you mention in the PR, the rDNS files needs to be handcrafted. Can this be automated to simplify the testing?